qb/delete_test.go

```diff
@@ -43,6 +45,30 @@ func TestDeleteBuilder(t *testing.T) {
 			S: "DELETE FROM cycling.cyclist_name WHERE id=? AND firstname>? ",
 			N: []string{"expr", "firstname"},
 		},
+		// Add tuple column
```

Comment:
What is the point in cloning all the tests but with tuple? Maybe one or two tests could be just fine?

Reply:
I was unsure of the code at that point. Perhaps you are right.